pkg/kubelet/dockershim/network/cni/cni_others.go

```diff
@@ -49,35 +53,55 @@ func getLoNetwork(binDirs []string) *cniNetwork {
 	return loNetwork
 }
 
-func (plugin *cniNetworkPlugin) platformInit() error {
-	var err error
-	plugin.nsenterPath, err = plugin.execer.LookPath("nsenter")
-	if err != nil {
-		return err
-	}
-	return nil
-}
-
 // TODO: Use the addToNetwork function to obtain the IP of the Pod. That will assume idempotent ADD call to the plugin.
 // Also fix the runtime's call to Status function to be done only in the case that the IP is lost, no need to do periodic calls
 func (plugin *cniNetworkPlugin) GetPodNetworkStatus(namespace string, name string, id kubecontainer.ContainerID) (*network.PodNetworkStatus, error) {
+	podIPs := plugin.getPodIPs(id)
+	if podIPs != nil && len(podIPs) > 0 {
+		klog.V(3).Infof("get pod ip %v from plugin", podIPs)
+		return &network.PodNetworkStatus{IP: podIPs[0], IPs: podIPs}, nil
+	}
+	cninetwork := plugin.getDefaultNetwork()
```

Comment:
Everything in this function from here on is concerned with fetching a result from CNI's cache; is that right?

I would probably comment that or create a new function with a name to indicate that.

Reply:
> Everything in this function from here on is concerned with fetching a result from CNI's cache; is that right?
> 
> I would probably comment that or create a new function with a name to indicate that.
yes, you're right. I agree it worth marking it.